client/state/notices/README.md

```diff
@@ -0,0 +1,53 @@
+Notices
+===========
+
+A module for managing global notices.
```

Comment:
I feel like the wording isn't quite right. Notices isn't really a single module... It's two modules... an actions module and a reducer module.